cmd/bb_storage/main.go

```diff
@@ -25,8 +33,12 @@ import (
 
 func main() {
 	var (
-		blobstoreConfig  = flag.String("blobstore-config", "/config/blobstore.conf", "Configuration for blob storage")
-		webListenAddress = flag.String("web.listen-address", ":80", "Port on which to expose metrics")
+		blobstoreConfig      = flag.String("blobstore-config", "/config/blobstore.conf", "Configuration for blob storage")
+		webListenAddress     = flag.String("web.listen-address", ":80", "Port on which to expose metrics")
+		agentEndpointURI     = flag.String("jaeger.agent-endpoint", "127.0.0.1:6831", "jaeger agent address")
+		collectorEndpointURI = flag.String("jaeger.collector-endpoint", "http://127.0.0.1:14268/api/traces", "jaeger collector endpoint")
+		serviceName          = flag.String("service-name", "bb_storage", "service name for tracing")
+		alwaysSample         = flag.Bool("always-sample", false, "record all traces. warning! do not use this in production!")
```

Comment:
Not sure if the warning is required, the [doc](https://godoc.org/go.opencensus.io/trace#AlwaysSample) already mentions that constantly sampling is likely to generate a lot of data, and in context, it is fairly easy to deduce that it might not be a good idea in a heavily used production service.